cmd/util/util.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"os"
+	"path/filepath"
+
+	"sigs.k8s.io/kustomize/kyaml/openapi"
+	"sigs.k8s.io/kustomize/kyaml/setters2"
+)
+
+// KRMFileName returns the name of the KRM file. KRM file determines package
+// boundaries and contains the openapi information for a package.
+var KRMFileName = func() string {
```

Comment:
Can't we use the value from `cmd/config'?

Reply:
Shouldn't we add cmd/config as a dependency for cli-utils if we want to use that value ?